learn/hoon/hoon-tutorial/setup.md

```diff
@@ -66,36 +40,15 @@ You just used a function from the Hoon standard library, `add`.  Next, quit Urbi
 $
 ```
 
-Your comet isn't running anymore and you're back at your computer's normal terminal prompt. You now have a `mycomet` directory for your Urbit comet; this directory is your comet's **pier** and it contains the entirety of your comet's state.
+Your ship isn't running anymore and you're back at your computer's normal terminal prompt. If your ship is `~palnul_nocser`, then you can restart the ship by typing:
 
-Right now the only thing in your pier is your comet's system files:
-
-```
-$ ls -a /path/to/mycomet
-./  ../ .urb/
-```
-
-Restarting your already-created comet from the terminal is like creating a new one, but without the `-c` flag.  (The `-c` is for "create".)
-
-```
-$ urbit mycomet
 ```
-
-Because the comet has already been booted it won't take very long to get it running again.  There are also fewer startup messages:
-
-```
-$ urbit mycomet
-[...]
-ames: on localhost, UDP 31337.
-http: live (insecure, public) on 8080
-http: live ("secure", public) on 8443
-http: live (insecure, loopback) on 12321
-~palnul_nocser:dojo>
+urbit palnul_nocser
```

Comment:
`palnul_nocser` is not a valid ship name (it's just a common way to abbreviate comet names), and comets always have user-defined names for their piers/directories, so this might be confusing.

If we're proceeding off the "development ship" instructions, probably just go with `~zod` in the prompt, and `my-fake-zod` as directory name (or whatever we commonly use in examples)?

Reply:
Ah, my mistake.